README.md

```diff
@@ -137,9 +137,8 @@ This project supports PHP 5.6 as minimum version (see [Drupal 8 PHP requirements
 To prevent this you can add this code to specify the PHP version you want to use in the `config` section of `composer.json`:
 ```json
 "config": {
-    "sort-packages": true,
     "platform": {
         "php": "5.6.40"
     }
-},
```

Comment:
"sort packages" is part of the provided composer.json. The example is correct. Please revert this.

Reply:
This snippet is focused on the "platform" setting AFAIK, so listing other allowed configurations here seems to bring no any useful context IMHO.

Reply:
> Instead of doing a lot of unrelated changes everywhere, including some that introduce errors, please do the simple one/two line change you really want please.
> 
> And please explain what is the problem with the current setup.

Thanks so much for the feedback.
I don't think the changes done in this PR are unrelated, that's why they are even grouped in the same commit.
Please, let me know if you have a different perspective about this in order to try a consensus about these changes.

Reply:
> This snippet is focused on the "platform" setting AFAIK, so listing other allowed configurations here seems to bring no any useful context IMHO.

Users would likely copy and paste this exact snippet into their `composer.json` file, which would lead to `sort-packages` being removed from their configuration

Reply:
>Users would likely copy and paste this exact snippet into their composer.json file, which would lead to sort-packages being removed from their configuration.

Based on this assumption, we should interpret [this snippet](https://github.com/drupal-composer/drupal-project/blob/ec0f41171516c18357ca4cda9068ee4cdeabaa19/README.md#should-i-commit-the-scaffolding-files) as a bad example too, didn't?
```yml
"scripts": {
    "post-install-cmd": [
        "@composer drupal:scaffold",
        "..."
    ],
    "post-update-cmd": [
        "@composer drupal:scaffold",
        "..."
    ]
},
```

Reply:
Probably. 😛 I suppose you're right. Maybe the docs should be updated to be more clear not to remove rules like this when you're updating your config, although that should probably be a separate PR.

Reply:
I've reverted this change, as it has nothing to do with the PR's goal. I agree that this topic should be addressed in a separate PR. Thank you so much for sharing your concerns about this.